src/shogun/preprocessor/RandomFourierGaussPreproc.h

```diff
@@ -99,17 +99,17 @@ class RandomFourierGaussPreproc: public RandomMixin<DensePreprocessor<float64_t>
 
 	/**  getter for the random coefficients
 	 * necessary for creating random fourier features compatible to the current ones
-	 * returns values of internal members randomcoeff_additive
-	 * and randomcoeff_multiplicative
+	 * returns values of internal members random_offset
+	 * and random_weights
 	 */
 	void get_randomcoefficients(float64_t ** randomcoeff_additive2,
```

Comment:
this should be refactored for SG* as well, rather than raw pointers

Reply:
True. I think it's inefficient to give and take pointers for the get method, maybe return a struct?

Reply:
do we even need this method? Users would not call it, but rather `get` using the appropriate names? For using the class, it is not needed

Reply:
and if it is an internal helper method, just use std::pair

Reply:
Facts, I think it's better to remove. Because we already have a way to extract internal members from other and appoint to the current(set_randomcoefficients), plus we have appropriate named getters anyway.